service/controller/resource/monitoring/scrapeconfigs/test/kvm/case-5-cluster-api-v1alpha3.golden

```diff
@@ -170,6 +170,7 @@
 
   tls_config:
     ca_file: /etc/prometheus/secrets/cluster-certificates/ca
+    
```

Comment:
Feels like we should have test with token auth mechanism somewhere.

Reply:
Tests added